airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/source.py

```diff
@@ -82,8 +82,9 @@ def check_connection(self, logger, config) -> Tuple[bool, any]:
         auth = self.get_authenticator(config)
         settings = None
         try:
+            datetime.strptime(config["start_date"], DATETIME_FORMAT)
             settings = UserSettingsStream(config["subdomain"], authenticator=auth, start_date=None).get_settings()
-        except requests.exceptions.RequestException as e:
+        except Exception as e:
```

Comment:
Could we be slightly more restrictive here and catch a datetime format issue in addition to the request exception?

To catch all exceptions could lead to some silent issues for customers.

Reply:
@bnchrch this is done on purpose. We want every exception to be caught, I would say this is a pattern for the `check` command implementation